README.md

```diff
@@ -1,70 +1,57 @@
-
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
-
-### Analyzing the Bundle Size
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
-
-### Making a Progressive Web App
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
-
-### Advanced Configuration
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+# Squizard's Spells
+
+### Mod3 Group Stretch Tech Project
+
+## Table of Contents
+- [Abstract](#Abstract)
+- [Technologies](#Technologies)
+- [Illustrations](#Illustrations)
+- [Install](#Install)
+- [Wins](#Wins)
+- [Challenges](#Challenges)
+- [Additions](#Additions)
+- [Contributors](#Contributors)
+
+## Abstract
+Squizard's Spells is an app that allows its users to filter through Dungeons and Dragons 5th Edition spells and save those spells in a personal spellbook. Spells saved will persist on page load as well. When a user reaches the site, they will be able to look at their personal spellbook or browse spells. Then they can choose the character class that they are playing to narrow it down to spells that that class can use. Players can then click on spells to get the full details or bookmark them to save them to their spellbook.
+
+## Technologies
+-  HTML
+-  CSS
+-  JavaScript
+-  React
```

Comment:
Maybe could add React-Router, REST Api, Cypress

Reply:
Yes, definitely want to add those!  I'd also add some of the specific hooks like 'useReducer' & 'useContext'.  Could mention the custom hooks too 🤓 🤓